package.json

```diff
@@ -38,17 +38,17 @@
     "lib": "./lib",
     "man": "./man"
   },
-  "main": "./lib/npm.js",
+  "main": "./lib/no-api.js",
   "bin": {
     "npm": "bin/npm-cli.js",
     "npx": "bin/npx-cli.js"
   },
   "exports": {
     ".": [
       {
-        "default": "./lib/npm.js"
+        "default": "./lib/no-api.js"
       },
-      "./lib/npm.js"
+      "./lib/no-api.js"
     ],
```

Comment:
you could also just remove the `.` section entirely, and modern node will error on `require('npm')`.

Reply:
This way the error shows that it's intentional, and explains when it was dropped.